### PR TITLE
feat(DTFS2-5659): added component tests for cookies, accessibility pages

### PR DESCRIPTION
**Pull request by vamshidharukef**
Introduction ✏️
Improve coverage for component tests for TFM UI

Resolution ✔️
Added more component tests for cookies, accessibility and change password page

### CODE REVIEW

**Review by lmarrai-sw — APPROVED** [review]

**Review by lmarrai-sw — APPROVED** [review]
Thanks for this @vamshidharukef

File: trade-finance-manager-ui/component-tests/cookies/cookies.component-test.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
+const pageRenderer = require('../pageRenderer');
+
+const page = '../templates/cookies.njk';
+
+const render = pageRenderer(page);
+
+describe(page, () => {
+  let wrapper;
+  const params = {
+    dealId: '1234',
```
Comment (avaitonis) [inline trade-finance-manager-ui/component-tests/cookies/cookies.component-test.js:10]: dealId parameter might be not required

**Review by avaitonis — DISMISSED** [review]
LGTM - with small advices for later

File: trade-finance-manager-ui/component-tests/user/change-password.component-test.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+const page = '../templates/user/change-password.njk';
+
+const render = pageRenderer(page);
+
+describe(page, () => {
+  let wrapper;
+  const params = {
+    dealId: '1234',
+  };
```
[the remark this answers is not in the document]
Comment (vamshidharukef) [reply trade-finance-manager-ui/component-tests/user/change-password.component-test.js:11]: Removed `dealId` param

**Review by avaitonis — APPROVED** [review]
LGTM - except one extra dealId

**Pull request merged**